_sass/_buttons.scss

```diff
@@ -10,7 +10,7 @@
 
   background-color: var(--_btn-background-color);
 
-  color: $navy;
+  color: var(--navy);
```

Comment:
Should this be set to the `$navy`?

Reply:
Yep, but I am changing how that is wired up with the addition of the text color control properties.